modules/shortcodes/email-subscribe.php

```diff
@@ -0,0 +1,233 @@
+<?php
+/**
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
+ **/
+
+/**
+ * Class Jetpack_Email_Subscribe
+ * This class encapsulates shortcode for subscribing to a MailChimp list.
+ * It displays a simple signup form that gets an email address from the user and signs him for a list
```

Comment:
```suggestion
 * The shortcode displays a simple signup form. Submitting an email address will sign the visitor up
```

Reply:
Thank you!

Reply:
I added a different comment that mentions gutenberg.